nurseconnect/templates/core/tags/topic_of_the_day.html

```diff
@@ -4,15 +4,15 @@
 {% if articles %}
     <section class="ContentList">
         <header class="ContentList-header">
-            <h1 class="ContentList-categoryLabel">{% trans "Topic of the day" %}</h1>
+            <h1 class="ContentList-categoryLabel">{% trans "Latest Articles" %}</h1>
```

Comment:
Ok, I'll move it to outside the topic of the day template. Maybe in the mani.html file?